### ISSUE TITLE
爱可可微博热门分享(3.2)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#91]
No 1. <a href="https://weibo.com/1402400261/O35Ifris6">#抽奖##赠书# 携手@异步图书 送出3本《ChatGPT原理与应用开发》</a>
No 2. <a href="https://weibo.com/1402400261/O388OxUjA">Open-Sora Plan：Sora开源复现项目</a>
No 3. <a href="https://weibo.com/1402400261/O31IxfqMM">oms-Diffusion：虚拟试穿项目，致力于解决服装行业高昂成本问题，支持IPadapter-faceID结合controlnet_openpose，用户可上传服装图片与参考姿势图进行试穿</a>
No 4. <a href="https://weibo.com/1402400261/O2X7KFVHk">#抽奖##赠书# 携手@图灵文化 @图灵新知 送出3本《大模型应用开发极简入门：基于GPT-4和ChatGPT》</a>
No 5. <a href="https://weibo.com/1402400261/O36Bx5uER">图机器学习课程资料</a>
No 6. <a href="https://weibo.com/1402400261/O35AdgvEm">[CL] Large Language Models and Games: A Survey and Roadmap 综述了大型语言模型(LLM)在游戏领域的应用现状，并展望了未来发展方向。文章指出，自2019年GPT-2发布以来，LLMs在文本生成的质量和控制性上取得了显著进步。本文强调了LLM在游戏内外可以扮演的多种角色，包括玩 ......</a>
No 7. <a href="https://weibo.com/1402400261/O31GqlcU9">face-to-sticker：把人脸转成贴画</a>
No 8. <a href="https://weibo.com/1402400261/O2VYRuRYi">#抽奖##赠书# 携手@博文视点Broadview 送出3本《图数据库：理论与实践》</a>
No 9. <a href="https://weibo.com/1402400261/O38p86HUE">luminal：一个深度学习库，利用可组合的编译器实现高性能。它以极快的速度运行，支持Metal和CUDA，使用Rust编写，直接与底层API交互，无需中间层。其核心思想是提前编译所有内容，采用静态计算图实现惰性执行，使得编译器可以全局优化，实现了高效的核心运算与编译时间的分离</a>
No 10. <a href="https://weibo.com/1402400261/O36mWzdjx">AI计算和环境之间的矛盾日益凸显</a>
No 11. <a href="https://weibo.com/1402400261/O2WPHlxyR">大语言模型将改变编程…但非常有限</a>
No 12. <a href="https://weibo.com/1402400261/O35wKr4eO">[CV] Trends, Applications, and Challenges in Human Attention ......</a>
No 13. <a href="https://weibo.com/1402400261/O32jkEsJK">晚安～ [月亮] ​</a>
No 14. <a href="https://weibo.com/1402400261/O2Pjk2wJQ">'青梧字幕 - 青梧字幕是一款基于whisper的AI字幕提取工具，使用C++版本的whisper.cpp进行底层处理，前端界面采用Electron ......</a>
No 15. <a href="https://weibo.com/1402400261/O2YJj4gl1">代表性LLM文本数据集大列表，包括预训练语料库、微调指令数据集、偏好数据集、评估数据集和传统NLP数据集</a>
No 16. <a href="https://weibo.com/1402400261/O35yAsr6g">[LG] StarCoder 2 and The Stack v2: The Next Generation 介绍了大型代码语言模型(Code ......</a>
No 17. <a href="https://weibo.com/1402400261/O35cjDp0O">[LG]《Beyond Language Models: Byte Models are Digital World Simulators》</a>
No 18. <a href="https://weibo.com/1402400261/O31UpwQKk">ActionWeaver：AI应用框架，使LLM函数调用更加简便</a>
No 19. <a href="https://weibo.com/1402400261/O2PmlmUcA">hotpdf：以快速、高效的方式解析PDF文件，可用于查找和提取PDF中的文本，克服使用pdfquery等库解析大型PDF文件时遇到的困难</a>
No 20. <a href="https://weibo.com/1402400261/O395y7dpF">几篇论文实现代码</a>
No 21. <a href="https://weibo.com/1402400261/O363LvDfl">马斯克起诉OpenAI：有悖初心</a>
No 22. <a href="https://weibo.com/1402400261/O35trbOjq">[CV] NARUTO: Neural Active Reconstruction from Uncertain Target ......</a>
No 23. <a href="https://weibo.com/1402400261/O35jM6lN0">[LG]《Training Dynamics of Multi-Head Softmax Attention for In-Context Learning: Emergence, Convergence, and Optimality》</a>
No 24. <a href="https://weibo.com/1402400261/O2Po5kQji">IntelliQ：基于大型语言模型(LLM)的多轮问答系统，结合了先进的意图识别和词槽填充(Slot Filling)技术，致力于提升对话系统的理解深度和响应精确度  </a>
No 25. <a href="https://weibo.com/1402400261/O3924cr31">’New API - 基于One API的二次开发版本' GitHub: github.com/Calcium-Ion/new-api ......</a>
No 26. <a href="https://weibo.com/1402400261/O2VBNcmip">[CL] Tokenization Is More Than Compression 网页链接 质疑了一个在自然语言处理(NLP)领域被广泛认同的假设：更少的分词(tokenization)结果会带来更好的下游任务表现。介绍了一种新的分词器PathPiece，旨在以最少的分词数量处理文本，并通过大量实验发现，减少分词数量并不一定能提升 ......</a>
No 27. <a href="https://weibo.com/1402400261/O31zlaMoG">MarauderMap：大规模勒索软件样本集</a>
No 28. <a href="https://weibo.com/1402400261/O2PsScr5v">𝚃𝚎𝚡𝚃𝚎𝚕𝚕𝚎𝚛：基于ViT的端到端公式识别模型，可以把图片转换为对应的latex公式</a>
No 29. <a href="https://weibo.com/1402400261/O2z9MDhG6">'《构建你自己的 AI 辅助编码助手》</a>
No 30. <a href="https://weibo.com/1402400261/O2YHOnGV8">The Foundation Model Development Cheatsheet：基础模型开发速查，旨在提供开发和发布模型的最佳实践。重点关注工具，如数据目录、搜索/分析工具、评估存储库，以及总结、调查或指导重要开发决策的文献</a>
No 31. <a href="https://weibo.com/1402400261/O2PljDR5t">'search2ai - 让你的 LLM API 支持联网，搜索、新闻、网页总结，已支持OpenAI、Gemini，大模型会根据你的输入判断是否联网' ......</a>
No 32. <a href="https://weibo.com/1402400261/O1TqzhLIA">2024年LLM研究趋势：模型合并，专家混合，更小的LLM</a>
No 33. <a href="https://weibo.com/1402400261/O35v5p1e1">[IR] Exploring the Impact of Large Language Models on Recommender ......</a>
No 34. <a href="https://weibo.com/1402400261/O1O1Fd62U">'收集C/C++新手学习的入门项目，整理收纳开发者开源的小项目、工具、框架、游戏等，视频，书籍，面试题/算法题，技术文章。' ......</a>
No 35. <a href="https://weibo.com/1402400261/O2VFIiaDJ">[CL] Stable LM 2 1.6B Technical Report 网页链接 StableLM 2 1.6B是一款新一代语言模型，特点是开源、透明且成本效益高，使用公开数据集进行训练，耗费约92k ......</a>
No 36. <a href="https://weibo.com/1402400261/O35aCEYKM">早！[太阳] ​</a>
No 37. <a href="https://weibo.com/1402400261/O2MsHA4qc">[CL] Latent Attention for Linear Time Transformers 提出一种新的Transformer模型'Latte ......</a>
No 38. <a href="https://weibo.com/1402400261/O2FOzpzJm">Mountaineer：旨在轻松用Python和R...</a>
No 39. <a href="https://weibo.com/1402400261/O2WKB3kvV">Google用Stack Overflow数据丰富其Gemini问答AI</a>
No 40. <a href="https://weibo.com/1402400261/O35deAMRe">[LG]《Griffin: Mixing Gated Linear Recurrences with Local Attention for Efficient Language Models》</a>
No 41. <a href="https://weibo.com/1402400261/O35EOjVWC">今日推介(第1332期)：用字节模型作为数字世界模拟器、基于混合门控线性递归与局部注意力的高效语言模型、神经网络可塑性丧失的原因分析、上下文学习中多头Softmax注意力的训练动态、基于梯度下降的关联记忆学习 ......</a>
No 42. <a href="https://weibo.com/1402400261/O2VrdmIcj">[LG]《Large Language Models As Evolution Strategies》</a>
No 43. <a href="https://weibo.com/1402400261/O2VBIDyZ3">通过统计物理方法分析了在高维空间和大数据集情况下，生成扩散模型的动态行为，揭示了从纯噪声到记忆化数据点的三个阶段，并通过对简化模型的分析和实际数据的验证，提供了一个理解维度诅咒和避免记忆化问题的理论框架。//[LG]《Dynamical Regimes of Diffusion Models》</a>
No 44. <a href="https://weibo.com/1402400261/O35cokZKg">提出bGPT模型，通过预测下一个字节的方法超越了传统深度学习的界限，能直接解读和操作二进制数据，实现对数字世界的内在和全面理解，不仅在传统领域(如文本、音频、图像)展现出色性能，也在模拟算法和硬件行为等新领域表现出创新能力和应用潜力。//[LG]《Beyond Language Models: Byte Models are Digital World Simulators》</a>
No 45. <a href="https://weibo.com/1402400261/O35F2FRWm">//@爱可可-爱生活：欢迎参与～</a>
No 46. <a href="https://weibo.com/1402400261/O35qTEEus">[LG]《Learning Associative Memories with Gradient Descent》</a>
No 47. <a href="https://weibo.com/1402400261/O2MoEaEpc">[LG] Does Negative Sampling Matter? A Review with Insights into ......</a>
No 48. <a href="https://weibo.com/1402400261/O35jQw6pm">提出了一个理论分析框架，用以研究多头Softmax注意力模型在多任务线性回归中上下文学习的梯度流训练动态，并首次证明了这一模型在适当初始化下的全局收敛性，揭示了训练过程中的“任务分配”现象，并证实了多头结构相较于单头结构具有明显的优势。//[LG]《Training Dynamics of Multi-Head Softmax Attention for In-Context Learning: Emergence, Convergence, and Optimality》</a>
No 49. <a href="https://weibo.com/1402400261/O2DOK7cVp">手把手教程：用Gemma, MongoDB 和开源模型构建RAG系统</a>
No 50. <a href="https://weibo.com/1402400261/O2z9i1em0">'本地LLM使用指南 0.2' GitHub: github.com/xue160709/Local-LLM-User-Guideline/blob/main/README-zh.md ......</a>
